events/using/aem-event-setup.md

```diff
@@ -228,25 +271,26 @@ To configure Adobe I/O Events as a cloud service in AEM:
 *   For **Organization ID**: Provide the ID shown on the Adobe I/O Console.
 *   For **Client Secret**: AEM will automatically retrieve the value from the Adobe I/O Console.
 
+
 ### Perform an AEM health and configuration check
```

Comment:
This part remains relevant to 6.4, right? The current formatting, makes it see only applicable to 6.2 and 6.3

Reply:
yes it's applicable to all